overlord/snapstate/autorefresh.go

```diff
@@ -203,6 +203,8 @@ func (m *autoRefresh) Ensure() error {
 	if CanAutoRefresh == nil {
 		return nil
 	}
+
+	// NOTE: this will unlock and re-lock state for network ops
```

Comment:
put here it seems to refer to CanAutoRefresh, but that one afaik is not doing network ops ?

Reply:
removed